openmetadata-airflow-apis/openmetadata_managed_apis/workflows/ingestion/common.py

```diff
@@ -99,8 +99,6 @@ def build_source(ingestion_pipeline: IngestionPipeline) -> WorkflowSource:
     if not service:
         raise ValueError(f"Could not get service from type {service_type}")
 
-    metadata.secrets_manager_client.add_service_config_connection(service, service_type)
```

Comment:
~~nice and clean~~ actually this we might need to remove at the very end, otherwise usage and profiler workflows might fail until we propagate the changes you just did

Reply:
We have done all the changes needed in the `Workflow` class so we can stop doing adding the service config connection to the service.